entries/l/latitudepay.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "LatitudePay": {
+    "domain": "latitudepay.com",
+    "contact": {
+      "facebook": "latitudefinancialservices",
```

Comment:
```suggestion
      "facebook": "LatitudePay",
```

Reply:
I would argue that the original one is the better option as it has more followers and is actually the verified account/promoted by the company

Reply:
On their [main page](https://www.latitudepay.com/), the linked Facebook account is "LatitudePay" - it doesn't get more official than that, I suppopse.

Reply:
Fair, you win this round